### DISCUSSION TITLE
CMT Paper Submission

### DISCUSSION THREAD

**Discussion by Vishruth-Sham** [FETS-AI/Challenge#112] [General]
I have initiated a submission on the CMT system, but I do not see any option to edit the submission. I thought we had time until 2nd August.
![image](https://user-images.githubusercontent.com/85173817/126363466-8c107762-fe3c-460a-a2e1-2e4f47f3ab9b.png)

**Comment by sarthakpati** [comment]
Tagging @ujjwalbaid0408

**Comment by ujjwalbaid0408** [comment]
Hi @Vishruth-Sham,

Can you check once again if you are able to edit the submission?

**Reply by Vishruth-Sham** [reply]
Yes, I can edit it now. Thanks!

**Comment by tphankr** [comment]
Please help me submit our results.
I registered and login on CMT but did not know how do chose any where in order to submit on CMT website. I chose as below following, is it righ? and the file is submitted should  or jpeg, jpg, png? 
![image](https://user-images.githubusercontent.com/37217354/126827886-d26c36e4-ffe2-47cd-9dbc-0a7e8f75c022.png)

![image](https://user-images.githubusercontent.com/37217354/126829369-82817194-b6f0-431a-9700-db201c1505ac.png)


Thank you.

**Reply by ujjwalbaid0408** [reply]
Hi, Please email challenge@fets.ai for this.

**Reply by tphankr** [reply]
@ujjwalbaid0408, Thank you.
Please help me check, I have just summitted as following. Is it right?
![image](https://user-images.githubusercontent.com/37217354/126829781-ccdc012d-3a54-4525-8a85-888424950f16.png)

And the end, we have like this and a email. 
![image](https://user-images.githubusercontent.com/37217354/126830125-be7ed3bc-204b-47a0-9738-42352386af63.png)


Did I submit yet?
thanks.

**Reply by ujjwalbaid0408** [reply]
Please write to challenge@fets.ai for any help regarding submission to IPP.
